h2o.xcodeproj/project.pbxproj

```diff
@@ -3638,11 +3638,11 @@
 				);
 				HEADER_SEARCH_PATHS = (
 					"$(inherited)",
-					"/usr/local/openssl-1.0.2/include",
+					/usr/local/openssl/include,
```

Comment:
Have you considered specifying the homebrew version (i.e. `/usr/local/opt/openssl/...`)?

If that works, then it might be a better choice, as that would align h2o.xcodeproj to the default setting CMakeLists.txt.

Reply:
Thank you. I did it in https://github.com/h2o/h2o/pull/2239/commits/b33d0c1631a053bfa2beac4d3c5a469dc0a6ea0d